patches/minecraft/net/minecraft/core/RegistryAccess.java.patch

```diff
@@ -1,11 +1,10 @@
 --- a/net/minecraft/core/RegistryAccess.java
 +++ b/net/minecraft/core/RegistryAccess.java
-@@ -185,6 +_,8 @@
-    private static <E> void m_211081_(RegistryResourceAccess.InMemoryStorage p_211082_, RegistryAccess.RegistryData<E> p_211083_) {
+@@ -186,6 +_,7 @@
        ResourceKey<? extends Registry<E>> resourcekey = p_211083_.f_123101_();
        Registry<E> registry = BuiltinRegistries.f_206379_.m_175515_(resourcekey);
-+      if (!resourcekey.equals(Registry.f_122818_))
-+         registry = ((Registry<Registry<E>>)BuiltinRegistries.f_123858_).m_6246_((ResourceKey<Registry<E>>)resourcekey);
```

Comment:
Why was this patch deleted? (Asking so it's documented here for future reference.)

Reply:
Per my commit description:
> Deleted hunk that is no longer needed - BuiltinRegistries.ACCESS already wraps around BuiltinRegistries.REGISTRY